### PR TITLE
Closes #361 | Create dataset loader for Thai-Lao Parallel Corpus

### LINKED ISSUE #361: Create dataset loader for Thai-Lao Parallel Corpus
Dataloader name: `tha_lao_embassy_parcor/tha_lao_embassy_parcor.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?tha_lao_embassy_parcor

|     Dataset| tha_lao_embassy_parcor |
|-------------|---|
| Description | Thai-Lao Parallel Corpus contains equivalent Thai and Lao sentence pairs derived from the website of the Royal Thai Embassy in Vientiane, Laos. |
| Subsets | - |
| Languages | tha, lao |
| Tasks | Machine Translation |
| License | Creative Commons Zero v1.0 Universal (cc0-1.0) |
| Homepage | https://github.com/PyThaiNLP/Thai-Lao-Parallel-Corpus/tree/master |
| HF URL | - |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by djanibekov**
Closes #361 
### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/tha_lao_embassy_parcor/tha_lao_embassy_parcor.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# TODO: Add BibTeX citation
```
Comment (MJonibek) [inline seacrowd/sea_datasets/tha_lao_embassy_parcor/tha_lao_embassy_parcor.py:27]:
```suggestion

```

**Review by MJonibek — APPROVED** [review]
@djanibekov, Everything works. Can you just remove one TODO on L27

**Comment by djanibekov** [comment]
noticed, done @MJonibek

**Review by holylovenia — APPROVED** [review]
LGTM!! Thanks for your hard work, @djanibekov. 🙏

**Pull request merged**